Decode Color and Type Abbreviations

macho-catt opened the pull request:
Fixes #259 

<details>
<summary>Before Change</summary>

![before](https://user-images.githubusercontent.com/48004150/133709728-8c05d6a6-50da-4220-b3e8-4d5747bc48b0.png)

</details>

<details>
<summary>After Change</summary>

![after](https://user-images.githubusercontent.com/48004150/133709736-c93ed944-6e1c-4624-af19-d620aaf5f8e8.png)

</details>

**Action Items**
- [x] Create type table based on the body_style that had the most counts in sql
- [x] Create color table based on the color that had the most counts in sql
- [x] If the two-letter abbreviation is unknown (not found in resources for body style and color codes / abbreviation is not obvious), 'Other' will be put instead

simzou commented on js/client/src/components/app/Sidebar.js:
```diff
@@ -100,11 +100,13 @@ function ConnectedSideBar({
           Type
-          <div className="leftData">{data.body_style}</div>
+          {/* If abbreviation is not in the table, use 'Other' */}
+          <div className="leftData">{T.typeTable[data.body_style] || "Other"}</div>
```
I know you're just using the existing import, but not a fan of `T` as a variable name here. Would prefer something more descriptive like `tables` Not a dealbreaker though.

simzou reviewed (dismissed):
Existing changes look good to me. 

The issue / UI slide isn't explicit about this, but these color and abbreviation decodings should be in the bar charts as well (i.e. the tables should be used in `Graph.js` as well). 

Around [Line 52](https://github.com/hackforla/lucky-parking/blob/6c26d581f6438911cf620c88739ef4c1f241f6ed/js/client/src/components/app/Graph.js#L52) is where some existing data processing is occurring so it could go there.

macho-catt commented:
I will work on incorporating those, thanks

macho-catt commented:
<details>
<summary>Updated Color Chart</summary>

![color_graph_new](https://user-images.githubusercontent.com/48004150/134103690-ebeff2ac-86ff-4db1-a555-147cd9937266.png)

</details>

<details>
<summary>Updated Body Style Chart</summary>

![style_graph_new](https://user-images.githubusercontent.com/48004150/134103680-61ee9d54-649d-4938-9a70-246e71166c0b.png)

</details>

**Action Items**
- [x] Changed T to tables to be more descriptive
- [x] Added decoding of color and body_style to charts
- [x] Changed if/else conditional to switch statement under Graph.js

qiqicodes reviewed (approved):
LGTM! 🙌
